### ISSUE TITLE
vis hangs on redirecting STDERR

### ISSUE THREAD

**Issue by Docbroke** [martanne/vis#985]
```
vis 2>/dev/null
```

**Comment by tosch42** [comment]
Hi,

vis prints its UI to `stderr` instead of `stdout`. By redirecting `stderr` to a file, you basically redirect vis' UI to the file. This can be reproduced by doing the following:
```sh
$ vis 2> log
$ cat log
```
You'll see a bunch of escape sequences which represent the UI.
On the other hand, redirecting `stdout` works perfectly:
```sh
$ vis > log
```

The only difference to vim here is that vim prints its UI to `stdout`. If you try the following
```sh
$ vim > log
```
vim will emit a warning and you got the same "problem".

TL;DR this is impossible to avoid with TUIs and not a real problem.

**Comment by Docbroke** [comment]
Thanks for your reply. So I just need to redirect stdout instead of stderr when trying to avoid error output.
